### PR TITLE
fix(nuxt): fallback to static error page on server error

### PR DESCRIPTION
**Pull request by antfu**
### 🔗 Linked issue

Part of #6683

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

When there is a server error, the `/__nuxt_error` might also fail to render, we fallback to use static HTML render in this case.

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: packages/nuxt/src/core/runtime/nitro/error.ts [hunk 1]
```diff
@@ -13,7 +14,7 @@ export default <NitroErrorHandler> async function errorhandler (error: H3Error,
     statusCode,
     statusMessage,
     message,
-    description: process.env.NODE_ENV === 'development' && statusCode !== 404
+    stack: process.env.NODE_ENV === 'development' && statusCode !== 404
```
Comment (antfu) [inline packages/nuxt/src/core/runtime/nitro/error.ts:17]: Not sure the reason initially, but this change makes the error page:

Before:
image

After:
image

Comment (pi0) [reply packages/nuxt/src/core/runtime/nitro/error.ts:17]: `description` field is used in error templates for title. I think we can now adjust it to error message for dev.

**Comment by pi0** [comment]
Trying locally, the production build errors makes an infinity loop. Investigating... 

image

File: packages/nuxt/src/core/runtime/nitro/error.ts [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import { withQuery } from 'ufo'
 import type { NitroErrorHandler } from 'nitropack'
 import type { H3Error } from 'h3'
-import { importModule } from '@nuxt/kit'
```
Comment (pi0) [inline packages/nuxt/src/core/runtime/nitro/error.ts]: @danielroe Do you have any idea why this wasn't caught by import protection?

**Review by pi0 — APPROVED** [review]

**Pull request merged**